src/client/hud.cpp

```diff
@@ -440,6 +440,7 @@ void Hud::drawLuaElements(const v3s16 &camera_offset)
 			case HUD_ELEM_IMAGE_WAYPOINT: {
 				if (!calculateScreenPos(camera_offset, e, &pos))
 					break;
+				[[fallthrough]];
```

Comment:
This is a syntax I have not seen before.

Reply:
Nice parts of the C++ syntax be like: `[[]][](){}()`